README.md

```diff
@@ -11,6 +11,7 @@ Available addons
 ----------------
 addon | version | summary
 --- | --- | ---
+[crm_action](crm_action/) | 10.0.1.0.0 | CRM Action
```

Comment:
This is changed automatically, so please don't change it by yourself or it will conflict.